specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1188,6 +1188,180 @@
+            },
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.\n\n * 500 InternalServerError - Something went wrong."
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Update tls version of the workspace managed sql server": {
+            "$ref": "./examples/UpdateWorkspaceManagedSqlServerDedicatedSQLminimalTlsSettings.json"
+          }
+        }
+      },
+      "get": {
+        "operationId": "WorkspaceManagedSqlServerDedicatedSQLminimalTlsSettings_Get",
+        "summary": "Get server's minimal tls settings.",
+        "description": "Get workspace managed sql server's minimal tls settings.",
+        "tags": [
+          "WorkspaceManagedSqlServer, DedicatedSQLminimalTlsSettings"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "name": "dedicatedSQLminimalTlsSettingsName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the dedicated sql minimal tls settings."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved dedicated sql minimal tls settings.",
+            "schema": {
+              "$ref": "#/definitions/DedicatedSQLminimalTlsSettings"
+            }
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.\n\n * 500 InternalServerError - Something went wrong."
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-long-running-operation": true,
```

Comment:
>"x-ms-long-running-operation": true [](http://example.com/codeflow?start=8&length=35)

GETs are not long running operations

Reply:
@pilor : Removed the long running operation in the latest commit. Please check